cratedb_toolkit/dms/table_mapping.py

```
@@ -0,0 +1,117 @@
+
+    def build(self) -> "TableMappingBuilder":
+        self.add_selection()
+        for name in self.names:
+            self.add_mapping(name)
+        return self
+
+    def render(self) -> str:
+        payload = {"rules": self.rules}
+        return json.dumps(payload, indent=4)
+
+    def add_rule(
```

Pylint (3.3.7)

[refactor] Too many arguments (8/5)

(R0913)